geopandas/sindex.py

```diff
@@ -438,6 +492,74 @@ def query_bulk(self, geometry, predicate=None, sort=False):
                 input_geometry_index.extend([i] * len(res))
             return np.vstack([input_geometry_index, tree_index])
 
+        def nearest(self, coordinates, num_results=1, objects=False):
+            """
+            Returns the nearest object or objects to the given coordinates.
+
+            Requires rtree, and passes parameters directly to
+            rtree.index.Index.nearest.
+
+            This behaviour is deprecated and will be updated to be consistent
+            with the pygeos PyGEOSSTRTreeIndex in a future release.
+
+            If longer-term compatibility is required, use rtree.index.Index.nearest
+            instead - this could be accessed by calling
+            super(type(s.sindex), s.index).nearest()
+
+            Examples
+            --------
+            >>> s = geopandas.GeoSeries(geopandas.points_from_xy(range(3), range(3)))
+            >>> s
+            0    POINT (0.00000 0.00000)
+            1    POINT (1.00000 1.00000)
+            2    POINT (2.00000 2.00000)
+            dtype: geometry
+
+            >>> list(s.sindex.nearest((0, 0)))  # doctest: +SKIP
+            [0]
+
+            >>> list(s.sindex.nearest((0.5, 0.5)))  # doctest: +SKIP
+            [0, 1]
+
+            >>> list(s.sindex.nearest((3, 3), num_results=2))  # doctest: +SKIP
+            [2, 1]
+
+            >>> list(super(type(s.sindex), s.sindex).nearest((0, 0),
+            ... num_results=2))  # doctest: +SKIP
+            [0, 1]
+
+            Parameters
+            ----------
+            coordinates : sequence or array
+                This may be an object that satisfies the numpy array protocol,
+                providing the index’s dimension * 2 coordinate pairs
+                representing the mink and maxk coordinates in each dimension
+                defining the bounds of the query window. For a TPR-Tree, this
```

Comment:
Remove the details for TPR-Tree (it isn't implemented according to the [docs](https://rtree.readthedocs.io/en/latest/class.html#rtree.index.Index.nearest)).